src/master-playlist-controller.js

```diff
@@ -915,26 +931,41 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     // Select a new playlist
     const nextPlaylist = this.selectPlaylist();
+
+    if (!nextPlaylist) {
+      this.error = 'Playback cannot continue. No available working or supported playlists.';
```

Comment:
outside the scope of this PR but we need to think about localizations from the strings in this project, particularly if they show errors to the user.